class/Traits/Database.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Trait for database operations.
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Traits;
+
+use BracketSpace\Notification\Utils\Cache\ObjectCache;
+
+/**
+ * Database trait
+ */
+trait Database {
```

Comment:
`Database` name is not good I think. It's too generic and this class could easily end up having >2k lines and 50 methods as almost all are related to DB. Better to name it `Users`